lib/Adapter/TolerantParser/Refactor/TolerantImportName.php

```diff
@@ -76,12 +76,17 @@ private function assertNotAlreadyImported(Node $node, NameImport $nameImport): v
         $currentClass = $this->currentClass($node);
         $imports = $node->getImportTablesForCurrentScope()[$this->resolveImportTableOffset($nameImport)];
 
-        if (null === $nameImport->alias() && isset($imports[$nameImport->name()->head()->__toString()])) {
-            throw new NameAlreadyImportedException($nameImport, $this->findExistingName($nameImport, $imports));
+        list($existingName, $existingImport) = $this->findExistingImport($nameImport, $imports);
```

Comment:
use `[ $existingName, $existingImport ] =`

Reply:
Added the `list_syntax` rule to the php cs fixer config.

Reply:
thanks - though in theory it will be overwritten with [this](https://github.com/phpactor/phpactor-hub/blob/master/templates/php-cs-fixer/php_cs.dist)

Reply:
Okay. Didn't know that. I could file a PR for the `hub` repo.

Reply:
See phpactor/phpactor-hub#1